src/components/VisualPickerOption/styled/content.js

```diff
@@ -1,8 +1,9 @@
 import styled from 'styled-components';
 import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
 
+const sizeMap = { large: '210px', medium: '142px', small: '100px' };
 const StyledContent = attachThemeAttrs(styled.span)`
-    height: 142px;
+    height: ${props => sizeMap[props.size] || '142px'};
```

Comment:
size is always defined because in defaultProps you put medium

Reply:
@yvmunayev
I think we need do a fallback to `|| '142px'` since the user can pass a wrong size value and the component should render nicely

Reply:
check here, I'm wrong and if necessary put:

`height: ${props => sizeMap[props.size] || '142px'};`